Troubleshooting/Observatory/remote-login.rst

```diff
@@ -0,0 +1,125 @@
+.. |author| replace:: *Wouter van Reeven*
+.. |contributors| replace:: **
+
+.. _remote-login:
+
+###############################
+Remote login to a CSC container
+###############################
+
+.. warning::
+
+    This procedure contains steps that may affect the system in critical ways.
+
+    If you are unsure about any of the steps described here, contact support personnel.
+
+
+Occasionally it may be necessary to test an update to the configuration of a component before tagging a new version of the configuration project.
+
+Most components on Rubin system run as containers on a Kubernetes cluster.
+Therefore, restarting the component after making changes will erase all changes.
+
+To connect to the container in the Kubernetes cluster, the ``kubectl`` command can be used.
+
+In order to do that, several items and pieces of information are needed:
+
+    * The KubeConfig file to connect to the kubernetes cluster
+    * The namespace in which the component has been deployed
+
+The KubeConfig file can be downloaded from rancher.
+For the application link, see :ref:`Observing-Interface-Operational-Environments`.
+In order to login, you need to use your IPA credentials.
+Once logged in, you see a list of the available kubernetes clusters.
+Click on the name of the cluster that you are trying to connect to.
+
+.. figure:: ./_static/rancher_dashboard.png
+    :name: fig-rancher-dashboard
+
+In the screenshot, you see the available clusters for the summit.
+In this case, you can connect to the ``yagan`` cluster.
```

Comment:
"In this case, you can connect to the ``yagan`` cluster." -> For the summit, you should connect to the ``yagan`` cluster.